plenum/server/domain_req_handler.py

```diff
@@ -56,8 +57,15 @@ def _reqToTxn(self, req: Request, cons_time):
 
     def apply(self, req: Request, cons_time: int):
         txn = self._reqToTxn(req, cons_time)
-        (start, end), _ = self.ledger.appendTxns(
```

Comment:
1) Why do pass a copy here?
2) Maybe it's better to get the second returned argument of `appendTxns` (which is ignored now), as this is the txn with seqNo applied?